File: pset1/mario/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // Prompt the user for the pyramid's height
    int height;
    do
    {
        height = get_int("Set pyramid's height (1-8): ");
    }
    while (height < 1 || height > 8);

    // Print a pyramid of that height
    for (int row = 0; row < height; row++)
    {
        for (int ls = 0; ls < height - row - 1; ls++)
        {
            printf(" ");
        }

        for (int lh = 0; lh < row + 1; lh++)
        {
            printf("#");
        }

        printf("  ");

        for (int rh = 0; rh < row + 1; rh++)
        {
            printf("#");
        }

        printf("\n");
    }
}

File: pset6/sentimental-mario-more/mario.py
from cs50 import get_int

# Prompt the user for the pyramid's height
while True:
    height = get_int("Set pyramid's height (1-8): ")
    if height >= 1 and height <= 8:
        break

# Print a pyramid of that height
for row in range(height):
    for ls in range(height - row - 1):
        print(" ", end="")
    for lh in range(row + 1):
        print("#", end="")
    print("  ", end="")
    for rh in range(row + 1):
        print("#", end="")
    print()
